.github/workflows/python_skbuild_wheels.yml

```diff
@@ -32,3 +32,38 @@ jobs:
         with:
           name: cibw-wheels-${{ matrix.os }}-${{ strategy.job-index }}
           path: ./wheelhouse/*.whl
+
+  build_wheels_win:
+    name: Build wheels on Windows
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install FFTW
+        run: |
+          Invoke-WebRequest -Uri "https://www.fftw.org/fftw-3.3.10.tar.gz" -OutFile "${{ github.workspace }}\fftw.tar.gz"
+
+          New-Item -Path "${{ github.workspace }}" -Name "fftw-source" -ItemType Directory
+          tar --strip-components=1 -C "${{ github.workspace }}\fftw-source" -zxf "${{ github.workspace }}\fftw.tar.gz"
```

Comment:
I think you could do steps up to here on unix and save the result.

Then you can run cmake install on all OSs and cache the result. Maybe the env variable contains Path/OSs and they can all find the library somewhere.

Reply:
> I think you could do steps up to here on unix and save the result.

So to have a workflow job that just downloads the FFTW source and saves it somewhere? Sure. But as it is, the FFTW source is only needed by Windows (macOS and Ubuntu can install the library and headers via package managers). Or maybe I'm missing something here?

Reply:
It is better to build fftw on all oses by hand as the package manager might ship a not vectorized version. I would like to have the fastest possible fftw bundled in finufft

Reply:
Good point. We should be able to copy the recipe here (maybe after we clean up a bit) to ubuntu/macos.